### ISSUE TITLE
My result is so bad, see my attachment

### ISSUE THREAD

**Issue by papayazm** [santi-pdp/segan#45]
Am I done something wrong? I download the latest seganv1.1 model, and use clean_wav.sh to clean  noise file. But the result is so bad that seems all is noise(see my attachments). The noisy_p277_111.wav is from the noisy_trainset_wav data, and the clean_p277_111.wav is my result.
Thanks
 
[clean_p277_111.zip](https://github.com/santi-pdp/segan/files/1883917/clean_p277_111.zip)
[noisy_p277_111.zip](https://github.com/santi-pdp/segan/files/1883918/noisy_p277_111.zip)

My Parsed arguments:  {'deconv_type': 'deconv', 'd_label_smooth': 0.25, 'z_depth':
256, 'preemph': 0.95, 'seed': 111, 'init_noise_std': 0.0, 'synthesis_path': 'dwa
vegan_samples', 'e2e_dataset': 'data/segan.tfrecords', 'save_freq': 50, 'g_type'
: 'ae', 'l1_remove_epoch': 150, 'epoch': 150, 'bias_D_conv': True, 'save_path':
'segan_allbiased_preemph', 'test_wav': './data/test1.wav', 'g_learning_rate': 0.
0002, 'unrolled_lstm': False, 'z_dim': 256, 'batch_size': 100, 'bias_downconv':
True, 'denoise_epoch': 5, 'canvas_size': 16384, 'noise_decay': 0.7, 'g_nl': 'pre
lu', 'beta_1': 0.5, 'd_learning_rate': 0.0002, 'denoise_lbound': 0.01, 'bias_dec
onv': True, 'weights': 'SEGAN-41700', 'model': 'gan', 'save_clean_path': './save
/', 'init_l1_weight': 100.0}

**Comment by MittalShruti** [comment]
Hi, how much data did you train your model on (if you remember..)
